cmd/singularity/build_test.go

```diff
@@ -90,16 +97,19 @@ func TestBuild(t *testing.T) {
 		buildSpec  string
 		sandbox    bool
 	}{
-		{"BusyBox", "", "../../examples/busybox/Singularity", false},
-		{"BusyBoxSandbox", "", "../../examples/busybox/Singularity", true},
-		{"Debootstrap", "debootstrap", "../../examples/debian/Singularity", true},
-		{"DockerURI", "", "docker://busybox", true},
-		{"DockerDefFile", "", "../../examples/docker/Singularity", true},
-		{"SHubURI", "", "shub://GodloveD/busybox", true},
-		{"SHubDefFile", "", "../../examples/shub/Singularity", true},
-		{"LibraryDefFile", "", "../../examples/library/Singularity", true},
-		{"Yum", "yum", "../../examples/centos/Singularity", true},
-		{"Zypper", "zypper", "../../examples/opensuse/Singularity", true},
+		{"BusyBox", "", "../../examples/busybox/Singularity", false, false, true},
```

Comment:
the type above (line 94) does not have three booleans, only one.

Also, is it necessary to put all of this in a single test? The three booleans at the end suggest there are three different tests here.

And this kind of test is more appropriate for E2E as this is not a unit test but a whole system test (it runs singularity with specific options).